app/facades/search_results.rb

```diff
@@ -0,0 +1,18 @@
+class SearchResults
+  def self.top_rated_movies
+    json = service.top_rated
+    json.map do |data|
+      movie(data)
+    end
```

Comment:
```suggestion
    movies = json.map do |data|
      movie(data)
    end
    movies.first(40)
```

We don't technically need the first 40 here but maybe we could add it as catch all just in case?

Reply:
The only issue with taking the first 40 is if there is a scenario in which you don't have 40 movies. If you are limiting this in your model anyway, don't duplicate the code. Just ensure that you have tests to check that the model is limiting the result to 40 as you expect.